### ISSUE TITLE
clear-select button inaccessible

### ISSUE THREAD

**Issue by sawyerclick** [rob-balfre/svelte-select#593]
The clear-select button using `pointerdown` instead of `click` makes the button inaccessible via a screenreader. Additionally, the button lacks a `title` or `aria-label` to signify its action
